skbio/alignment/_repr.py

```diff
@@ -0,0 +1,66 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, scikit-bio development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+from __future__ import absolute_import, division, print_function
+
+from skbio.util._metadata_repr import _MetadataReprBuilder
+
+
+class _TabularMSAReprBuilder(_MetadataReprBuilder):
+
+    def _process_header(self):
+        cls_name = self._obj.__class__.__name__
+        if self._obj.dtype is not None:
+            dtype_class = '<' + self._obj.dtype.__name__ + '>'
+        else:
+            dtype_class = ''
+        self._lines.add_line(cls_name + dtype_class)
+        self._lines.add_separator()
+        self._ellipse_insert = ' ... '
```

Comment:
It's a little odd to set `_ellipse_insert` here, I think this makes more sense in the constructor